package.json

```diff
@@ -108,6 +108,9 @@
     "express": "^4.17.1",
     "express-promise-router": "^4.1.0",
     "express-validator": "^6.12.0",
+    "googleapis": "^67.1.1",
+    "js-base64": "^3.7.2",
```

Comment:
I don't see where this is used - presumably in an upcoming PR? If it's in EmailUtils already then this dependency should have been included in the previous PE. Only include changes relevant to this PR. Same goes for jsonwebtoken and googleapis - they should only be included in a PR that uses them

Reply:
The dependency googleapis should have been included in the previous PR and is required for the sendEmail function in EmailUtils.  

I uninstalled js-base64 and jsonwebtoken which will be included in a future PR.